detect_res[] det_list